templates/diffusion/run_qunex.jinja2

```diff
@@ -10,6 +10,23 @@
     #json_location=fd -a -e json dMRI ~/XNAT_BUILD_DIR/HCPpipeline/diffusion_Q3_2020_runs/ | rg -v SBRef | head -1
     json_file=`find "${StudyFolder}" -iname "*dMRI*.json" | grep -v "SBRef" | head -1`
     echo "json_file=${json_file}"
+
+
+    spacing=`python << END
+    import json
+    json_file = "$json_file"
+    with open(json_file) as f:
+        data = json.load(f)
+        spacing = data["EffectiveEchoSpacing"]
+        # convert from seconds to milliseconds
+        spacing *= 1000
+        print(spacing)
+    END`
+
+    echo "# The was extracted from `EffectiveEchoSpacing` in $json_file." >> ${StudyFolder}/processing/batch.txt
```

Comment:
```suggestion
    echo "# Value for the following was extracted from `EffectiveEchoSpacing` in $(basename $json_file)." >> ${StudyFolder}/processing/batch.txt
```
Also, $json_file is going to be some long absolute path.  Thus the proposal to use `basename` to extract just the filename for purposes of what goes into the batch.txt comment.

Reply:
FWIW, it's cool if you like python, but `grep` piped to `cut` (or `awk`) would be a one-liner.  :)

Reply:
Just also realized, I think you need single quotes around `EffectiveEchoSpacing`, rather than back-tics.  o.w., the shell is going to try to evalulate it as a command.